rpm/0008-Retrieve-message-lists-based-on-the-folder-sync-poli.patch

```diff
@@ -8,16 +8,88 @@ when downloading messages from different folders.
 
 Also captures the sync policy as part of the QMailAccount structure.
 ---
- src/libraries/qmfclient/qmailaccount.cpp      | 10 ++++
- src/libraries/qmfclient/qmailaccount.h        | 10 ++++
- src/libraries/qmfclient/qmailstore_p.cpp      | 47 +++++++++++++++++++
- src/libraries/qmfclient/qmailstore_p.h        |  3 ++
+ src/libraries/qmfclient/libaccounts_p.cpp     | 41 +++++++++++++++++
+ src/libraries/qmfclient/qmailaccount.cpp      | 10 +++++
+ src/libraries/qmfclient/qmailaccount.h        | 10 +++++
  .../messageservices/imap/imapservice.cpp      | 21 ++++++---
- .../messageservices/imap/imapstrategy.cpp     | 45 ++++++++++++++++--
+ .../messageservices/imap/imapstrategy.cpp     | 45 +++++++++++++++++--
  .../messageservices/imap/imapstrategy.h       |  7 +--
  .../messageservices/pop/popservice.cpp        |  7 +++
- 8 files changed, 138 insertions(+), 12 deletions(-)
+ 7 files changed, 129 insertions(+), 12 deletions(-)
 
+diff --git a/src/libraries/qmfclient/libaccounts_p.cpp b/src/libraries/qmfclient/libaccounts_p.cpp
+index 88d882e4..a5eded66 100644
+--- a/src/libraries/qmfclient/libaccounts_p.cpp
++++ b/src/libraries/qmfclient/libaccounts_p.cpp
+@@ -482,6 +482,42 @@ bool SSOAccountSatisfyTheKey(Accounts::Account* ssoAccount, const QMailAccountKe
+     return false;
+ }
+ 
++QMailAccount::FolderSyncPolicy folderSyncPolicyToEnum(QString const &policy)
```

Comment:
Seems just moving code around, but more commonly these would be const QString &. No need to change, but wouldn't mind if done.

Reply:
I think the code move comes from the regeneration of the patch with git format-patch now that the patch is applied to the file libaccount_p.cpp and not qmailstore_p.cpp anymore.

I've change the declaration to be const QString &.